jumpcloud-ADMU/Powershell/Tests/manualTests/selectionForm.feature

```diff
@@ -0,0 +1,217 @@
+        When a domain profile is selected
+        And a JumpCloud valid username is specified in the text box
+        And a password "" is specified in the text box
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+Feature: Selection form disallows user migration for invalid APIKeys
+
+    Scenario: Disallows "Migrate Profile" Button for null APIKeys
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "AutoBind JCUser" checkbox is selected
+        And a "" is pasted in the APIKey text field
+        And a JumpCloud valid username is specified in the username text box
+        And a valid password is specified in the password text box
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+Feature: Selection form disallows user migration for invalid ConnectKeys
+
+    Scenario: Disallows "Migrate Profile" Button for null ConnectKeys
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "Install JCAgent" checkbox is selected
+        And a "" is pasted in the ConnectKey text field
+        And a JumpCloud valid username is specified in the username text box
+        And a valid password is specified in the password text box
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+    Scenario: Disallows "Migrate Profile" Button for non-40 character ConnectKeys
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "Install JCAgent" checkbox is selected
+        And a "1234" is pasted in the ConnectKey text field
+        And a JumpCloud valid username is specified in the username text box
+        And a valid password is specified in the password text box
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+    Scenario: Disallows "Migrate Profile" Button for ConnectKeys with spaces
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "Install JCAgent" checkbox is selected
+        And a "11111111111111111111 1111111111111111111" is pasted in the ConnectKey text field
+        And a JumpCloud valid username is specified in the username text box
+        And a valid password is specified in the password text box
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+Feature: JumpCloud Username & AutoBind User Validation
+
+    Scenario: Disallows migration for usernames that are not found in the JumpCloud organization
```

Comment:
Can we not just disable the Migrate Profile button? 
<img width="770" alt="image" src="https://github.com/user-attachments/assets/64886d6b-f857-43e5-a4ab-26e9104e3d8b" />

Reply:
In my opinion, I'm wanting to move some things to a new validate button. We'd be putting a lot of code into the Test-MigrateButton if we also validate the agent. I'm not opposed to it, but I've already changed so much of the form in this release, I'm not wanting to consolidate more.

Reply:
Agree, we can create another to look into this